### PR TITLE
VZ-8735 - Restructured

### PR DESCRIPTION
**Pull request by MelwynPaul**
- Created an Observability section
- Created Monitoring and Logging sub-sections
- Created/moved monitoring-related topics
- Logging > For now we have Logging and Tracing info under this (to split this section - later)

### CODE REVIEW

File: content/en/docs/observability/monitoring/configure-metrics.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Metrics"
-linkTitle: Metrics
+title: "Monitoring Components in Verrazzano"
```
Comment (rosemarymarano) [inline content/en/docs/observability/monitoring/configure-metrics.md:2]: -> Monitor Components in Verrazzano (same for link title)

File: content/en/docs/observability/monitoring/metrics/_index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: "Configure Monitoring Components"
+linkTitle: Configure
+description: "Learn how to configure various monitoring components"
```
Comment (rosemarymarano) [inline content/en/docs/observability/monitoring/metrics/_index.md:4]: -> Learn how to configure monitoring components

File: content/en/docs/observability/monitoring/metrics/configure/prometheus/prometheus.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
 ---
-title: Prometheus
-description: Customize Verrazzano Prometheus settings
+title: "Prometheus"
 linkTitle: Prometheus
-Weight: 12
+description: "Learn how to use customize Prometheus to monitor Verrazzano"
```
Comment (rosemarymarano) [inline content/en/docs/observability/monitoring/metrics/configure/prometheus/prometheus.md:4]: -> Learn how to use Prometheus to monitor Verrazzano or Learn how to customize Prometheus to monitor Verrazzano

**Review by rosemarymarano — COMMENTED** [review]
Few edits.

File: content/en/docs/observability/monitoring/configure-metrics.md (hunk 1 above)
Comment (MelwynPaul) [reply content/en/docs/observability/monitoring/configure-metrics.md]: Thank you. 
Updated the title to: Monitor Components in Verrazzano
Link text to: Monitor Components

File: content/en/docs/observability/monitoring/metrics/_index.md (hunk 2 above)
Comment (MelwynPaul) [reply content/en/docs/observability/monitoring/metrics/_index.md]: Thank you. 
Updated.

**Review by rosemarymarano — APPROVED** [review]

File: content/en/docs/observability/monitoring/metrics/configure/prometheus/prometheus.md (hunk 3 above)
Comment (MelwynPaul) [reply content/en/docs/observability/monitoring/metrics/configure/prometheus/prometheus.md]: Ah... missed deleting "use" 
Updated. Thank you.

**Pull request merged**